### PR TITLE
Websocket virtctl

### PR DESCRIPTION
**Pull request by rmohr**
When the VM is running, run 

```
cluster/kubectl.sh console testvm
```

### CODE REVIEW

File: cmd/virtctl/virtctl.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	if err != nil {
+		log.Fatal("Make raw terminal:", err)
+	}
+	defer terminal.Restore(int(os.Stdin.Fd()), state)
+	fmt.Fprint(os.Stderr, "Escape sequence is ^]")
+
+	go func() {
+		defer close(readStop)
+		for {
+			_, message, err := c.ReadMessage()
```
Comment (stu-gott) [inline cmd/virtctl/virtctl.go]: This looks like it's expecting an "eof" type of error, but c is a general DefaultDialer. Are there any other error conditions to be worried about?

File: pkg/virtctl/console/console.go [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+	cf := flag.NewFlagSet("console", flag.ExitOnError)
+	cf.StringP("device", "d", "serial0", "Console to connect to")
+
+	return cf
+}
+
+func (c *Console) Usage() string {
+	usage := "Connect to a serial console on a VM:\n\n"
+	usage += "Examples:\n"
+	usage += "#Connect to the console 'serial0' on the VM 'myvm':\n"
```
Comment (stu-gott) [inline pkg/virtctl/console/console.go]: Was the "#" character intentional?

Comment (rmohr) [inline pkg/virtctl/console/console.go]: Yes, but a whitespace was missing. Have for instance a look at `kubectl get --help` to see how they are formatting the help pages.

File: cluster/kubectl.sh [hunk 3]
```diff
@@ -22,6 +22,11 @@ if [ "x$1" == "xspice" ]; then
         exit
 fi
 
+if [ "x$1" == "xconsole" ]; then
+    cmd/virtctl/virtctl -s http://${master_ip}:8184 "$@"
+    exit
+fi
```
Comment (rmohr) [inline cluster/kubectl.sh]: Done

**Comment by fabiand** [comment]
Rephrase: A second reviewer should ack.

**Comment by rmohr** [comment]
retest this please

**Comment by rmohr** [comment]
retest this please

**Comment by rmohr** [comment]
retest this please

**Pull request merged**